scripts/build/unzip.ps1

```diff
@@ -1,15 +1,17 @@
 <#
     .SYNOPSIS
     This script wraps c#'s io.compression.zipfile to unzip a zip file.
-    The script is intended to be used within the VSO build to unzip the winobjc SDK.
+    The script is intended to be used within the VSO build to unzip a NuGet package.
 #>
 param(
-    #NOTE: using Parameter(Mandatory) will hang the VSO build job since powershell will block waiting for user input.
+    #NOTE: using Parameter(Mandatory) will hang the VSO build job since
+    #      powershell will block waiting for user input.
     [string]$zipFile = $(throw 'Mandatory parameter "zipFile" is not set.'),
```

Comment:
We will need to "install" the tools package. I thought it would be better for build isolation to not actually install it and instead just unzip the package and run vsimporter from the unzipped package